### PR TITLE
[misc-] fix minor typos, and add-rows prompt

### PR DESCRIPTION
**Pull request by midichef**
Three minor commits:

1. The first commit fixes a few self-explanatory typos.

2. The `add-rows` prompt (`add rows: 1`) is a bit vague. The new wording makes it just a bit clearer that what the user is inputing is a count of rows. Otherwise, they may think they're being prompted for a row index. Or they just may not understand what is being asked.

### CODE REVIEW

File: visidata/modify.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def rowDeleted(self, row):
     self.unselectRow(row)
     def _undoRowDeleted(sheet, row):
         if sheet.rowid(row) not in sheet._deferredDels:
-            vd.warning('cannot undo to before commit')
+            vd.warning('cannot undo before commit')
```
Comment (saulpw) [inline visidata/modify.py]: This is/was inferring that the user was trying to undo a row deletion after a commit operation, which is not possible.  So I agree the message should be changed, but as stated now is incorrect.

File: visidata/modify.py [hunk 2]
```diff
@@ -330,7 +330,7 @@ def new_rows(sheet, n):
     return [sheet.newRow() for i in range(n)]
 
 Sheet.addCommand('a', 'add-row', 'addRows([newRow()], index=cursorRowIndex); cursorDown(1)', 'append a blank row')
-Sheet.addCommand('ga', 'add-rows', 'n=int(input("add rows: ", value=1)); addRows(new_rows(n), index=cursorRowIndex); cursorDown(1)', 'append N blank rows')
+Sheet.addCommand('ga', 'add-rows', 'n=int(input("add how many rows: ", value=1)); addRows(new_rows(n), index=cursorRowIndex); cursorDown(1)', 'append N blank rows')
```
Comment (saulpw) [inline visidata/modify.py]: `add # rows: ` ?

**Review by saulpw — APPROVED** [review]
Thanks for this attention to detail, @midichef !

File: visidata/modify.py (hunk 1 above)
Comment (midichef) [reply visidata/modify.py]: Oops, I did not read the context. How about, as you said, `cannot undo row deletion after a commit`?

There are two more places in this file to change the warning as well, `rowAdded()`, and `cellChanged()`.

File: visidata/modify.py (hunk 2 above)
Comment (midichef) [reply visidata/modify.py]: That's better: clearer and shorter.

File: visidata/modify.py (hunk 1 above)
Comment (saulpw) [reply visidata/modify.py]: Yes, that wording sounds fine to me.  Thanks, @midichef.

**Pull request merged**